server/src/test/java/org/elasticsearch/search/aggregations/bucket/nested/ReverseNestedAggregatorTests.java

```diff
@@ -240,13 +238,7 @@ public void testFieldAlias() throws IOException {
         }
     }
 
-    /**
-     * {@link NumericTermsAggregator} is the first complex bucking aggregation
-     * that stopped wrapping itself in {@link AggregatorFactory#asMultiBucketAggregator}
-     * so this tests that nested works properly inside of it.
-     */
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/56529")
```

Comment:
I already fixed this but somehow didn't remove this! It passed 1000 times for me though.

Reply:
💯 💯 💯